src/Command/Docker/Initialize.php

```diff
@@ -58,7 +58,7 @@ protected function configure()
         $this->setName('docker:initialize')->setDescription('Initialize the project and all the services.');
         $this->setAliases(['docker:init', 'initialize', 'init']);
         $this->addArgument('repository', InputArgument::OPTIONAL, 'eZ Platform Repository', 'ezsystems/ezplatform');
-        $this->addArgument('version', InputArgument::OPTIONAL, 'eZ Platform Version', '1.9.*');
+        $this->addArgument('version', InputArgument::OPTIONAL, 'eZ Platform Version', '1.11.*');
```

Comment:
Ok if you update this to 1.12 in a few days :)
Did you test against the rc btw?

Reply:
yes all good!